### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**
Cart Done

### CODE REVIEW

File: components/Cart.css [hunk 1]
```diff
@@ -73,7 +72,6 @@
 
 .summary {
   margin-top: 12rem;
-  /* margin-left: 2rem; */
   margin-left: -5rem;
```
Comment (ArjunJunna) [inline components/Cart.css:75]: You can use margin shorthand property here.

**Review by ArjunJunna — APPROVED** [review]

File: Pages/cart.html [hunk 2]
```diff
@@ -93,7 +93,7 @@ <h3>Ice-cream</h3>
 
 
         <div class="filters summary">
-            <span style="font-size: 3rem;">Subtotal (3) items</span>
+            <span style="font-size: 2rem;">Subtotal (3) items</span>
```
Comment (ArjunJunna) [inline Pages/cart.html:96]: Can avoid using inline styling.

**Review by ArjunJunna — APPROVED** [review]

File: components/Cart.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+.cart-btn {
+  background-color: var(--text-white);
+  border: none;
+  color: var(--primary-dark-color);
+  margin-top: 3rem;
+  height: 5rem;
+  border-radius: 0.5rem;
+  font-size: 1.8rem;
+  margin-top: 1rem;
```
Comment (ArjunJunna) [inline components/Cart.css:87]: Can remove one margin-top.

**Review by ArjunJunna — APPROVED** [review]

**Review by MayankKumar10 — APPROVED** [review]
you can use the shorthand property of margin and padding (top, right, bottom, left), you can use this in that order. Everything else is fine you can merge it

File: components/Cart.css (hunk 1 above)
Comment (MuskanChhatrasal) [reply components/Cart.css:75]: Okayy!! Will do that.

File: Pages/cart.html (hunk 2 above)
Comment (MuskanChhatrasal) [reply Pages/cart.html:96]: Okayy!! Will avoid using it.

File: components/Cart.css (hunk 3 above)
Comment (MuskanChhatrasal) [reply components/Cart.css:87]: Done!! Thankyou for pointing it.

**Comment by MuskanChhatrasal** [comment]
Okayy!! Will do that.

**Pull request merged**